operators/3scale-community-operator/0.11.0/bundle.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=3scale-community-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=threescale-2.14
+LABEL operators.operatorframework.io.bundle.channel.default.v1=threescale-2.14
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.2.0
+LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
+LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v2
+LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
+LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
+LABEL com.redhat.openshift.versions=v4.9
```

Comment:
is this correct? We want this to be available on 4.9?

Reply:
I removed this line: LABEL com.redhat.openshift.versions=v4.9. It doesn't exist in 3scale-operator bundle.Dockerfile. 
Thank you